tensorflow/lite/micro/kernels/reduce_common.cc

```diff
@@ -120,7 +120,7 @@ enum MinMaxEvalType { kEvalMin, kEvalMax };
 template <typename T>
 struct MinMaxReducerCompare {
   MinMaxReducerCompare() = delete;
-  MinMaxReducerCompare(MinMaxEvalType evalType) : type_(evalType){};
+  MinMaxReducerCompare(MinMaxEvalType evalType) : type_(evalType) {};
```

Comment:
I believe the Google style rules say that a space is allowed between the parenthesis and the brace, and my IDE with it's auto formatting agrees.  But the Pigweed style checker says that this is wrong and no spacing can be there.  I am much interested in your final resolution of this.

Reply:
I'm sticking with the current format because it follows the rule "There should be a space between the close parenthesis and the open curly brace." ([link](https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions) 

Given that the Pigweed style is implemented by clang-format (following Google conventions), the practical decision is to treat the tool's output as the final authority, overriding minor written style guide disagreements.